tests/utils/decorators/test_decorators.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Import modules
+import pytest
+import numpy as np
+from numbers import Number
+
+# Import from package
+from pyswarms import cost
+
+
+@pytest.mark.parametrize(
+    "objective_func",
+    [np.sum(), np.prod()]
+)
+def test_cost_decorator(objective_func, particles):
+    def cost_func_w_o_decorator(x):
```

Comment:
It's ok to spell it out :laughing: `cost_func_without_decorator(x)` and `cost_fun_with_decorator`